### PR TITLE
Add CR3 code sheet link to support menu

### PR DESCRIPTION
**Pull request by patrickm02L**
## Associated issues

This issue closes [#11799](https://github.com/cityofaustin/atd-data-tech/issues/11799)

VZ Staff wanted to add a link to the support menu that linked to the CR3 codes. 

## Testing
**URL to test:** 
https://deploy-preview-1216--atd-vze-staging.netlify.app/#/dashboard

**Steps to test:**
- Click on Support Menu (?) to see new link to CR3 code sheet
- Click on link to make sure it goes to [external link](https://ftp.dot.state.tx.us/pub/txdot-info/trf/crash_notifications/cr3_code_sheet.pdf)

---
#### Ship list
- [ ] Code reviewed
- [ ] Product manager approved

### CODE REVIEW

**Review by mddilley — APPROVED** [review]
Tested this out with the deploy preview, and it works great! 🚀 Good to know that this document exists too!

File: atd-vze/src/containers/DefaultLayout/DefaultHeader.js [hunk 1]
```diff
@@ -98,6 +98,12 @@ const DefaultHeader = props => {
               >
                 <i class="fa fa-wrench" /> Request an enhancement&nbsp;&nbsp; <i className="fa fa-external-link" />
               </DropdownItem>
+              <DropdownItem
+                href="https://ftp.dot.state.tx.us/pub/txdot-info/trf/crash_notifications/cr3_code_sheet.pdf"
+                target="_blank"
```
Comment (chiaberry) [inline atd-vze/src/containers/DefaultLayout/DefaultHeader.js:103]: I remembered seeing a new cheat sheet in the april release, and started looking through the url folder that this one was in and found this link
https://ftp.dot.state.tx.us/pub/txdot-info/trf/crash_notifications/2023/code-sheet.pdf

**Review by chiaberry — APPROVED** [review]
i like calling the menu support over help -- thanks for updating that. 

Left a comment about the updated code sheet.

**Review by frankhereford — APPROVED** [review]
Thanks @chiaberry for finding the updated version of this. Bummer they don't keep a "-latest.pdf" URL the the most recent one available for sites to link to. 

With the new URL that Chia found, this is perfect. Thanks so much for updating this!

File: atd-vze/src/containers/DefaultLayout/DefaultHeader.js (hunk 1 above)
Comment (patrickm02L) [reply atd-vze/src/containers/DefaultLayout/DefaultHeader.js]: Yes, awesome. I updated the link to point to this most recent cheat sheet. Also, sent a quick note asking him if that was okay. He thanked us for using the most updated version.

**Pull request merged**